<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BossGoBuy</class>
 <widget class="QWidget" name="BossGoBuy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>602</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1162</width>
    <height>602</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="4,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>30</y>
              <width>80</width>
              <height>70</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/icon/images/icon/cake.svg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>440</x>
              <y>30</y>
              <width>80</width>
              <height>70</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/icon/images/icon/Sauce.svg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>820</x>
              <y>20</y>
              <width>80</width>
              <height>70</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/icon/images/icon/Cilantro.svg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>180</y>
              <width>80</width>
              <height>70</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/icon/images/icon/Egg.svg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>440</x>
              <y>170</y>
              <width>80</width>
              <height>70</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/icon/images/icon/Potato.svg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>830</x>
              <y>170</y>
              <width>80</width>
              <height>70</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/icon/images/icon/Crispbread.svg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>320</y>
              <width>80</width>
              <height>70</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/icon/images/icon/Ham.svg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>440</x>
              <y>310</y>
              <width>80</width>
              <height>70</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/icon/images/icon/loin.svg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_11">
            <property name="geometry">
             <rect>
              <x>830</x>
              <y>310</y>
              <width>85</width>
              <height>75</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/icon/images/icon/Youtiao.svg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_12">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>110</y>
              <width>71</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;宋体&quot;;</string>
            </property>
            <property name="text">
             <string>原味饼</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_13">
            <property name="geometry">
             <rect>
              <x>470</x>
              <y>100</y>
              <width>31</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;宋体&quot;;</string>
            </property>
            <property name="text">
             <string>酱</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_14">
            <property name="geometry">
             <rect>
              <x>840</x>
              <y>100</y>
              <width>71</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;宋体&quot;;</string>
            </property>
            <property name="text">
             <string>香菜</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_15">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>260</y>
              <width>71</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;宋体&quot;;</string>
            </property>
            <property name="text">
             <string>鸡蛋</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_16">
            <property name="geometry">
             <rect>
              <x>450</x>
              <y>250</y>
              <width>71</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;宋体&quot;;</string>
            </property>
            <property name="text">
             <string>土豆丝</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_17">
            <property name="geometry">
             <rect>
              <x>850</x>
              <y>250</y>
              <width>71</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;宋体&quot;;</string>
            </property>
            <property name="text">
             <string>脆饼</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_18">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>410</y>
              <width>71</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;宋体&quot;;</string>
            </property>
            <property name="text">
             <string>火腿肠</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_19">
            <property name="geometry">
             <rect>
              <x>450</x>
              <y>410</y>
              <width>71</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;宋体&quot;;</string>
            </property>
            <property name="text">
             <string>里脊</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_20">
            <property name="geometry">
             <rect>
              <x>850</x>
              <y>400</y>
              <width>71</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;宋体&quot;;</string>
            </property>
            <property name="text">
             <string>油条</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="OriginalCakeSpinBox">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>50</y>
              <width>141</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
           <widget class="QSpinBox" name="EggSpinBox">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>190</y>
              <width>141</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="HamSpinBox">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>330</y>
              <width>141</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="SauceSpinBox">
            <property name="geometry">
             <rect>
              <x>550</x>
              <y>50</y>
              <width>141</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="PotatoSpinBox">
            <property name="geometry">
             <rect>
              <x>550</x>
              <y>190</y>
              <width>141</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="LoinSpinBox">
            <property name="geometry">
             <rect>
              <x>550</x>
              <y>320</y>
              <width>141</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="CilantroSpinBox">
            <property name="geometry">
             <rect>
              <x>930</x>
              <y>40</y>
              <width>141</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="CrispbreadSpinBox">
            <property name="geometry">
             <rect>
              <x>930</x>
              <y>180</y>
              <width>141</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="YoutiaoSpinBox">
            <property name="geometry">
             <rect>
              <x>930</x>
              <y>320</y>
              <width>141</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="labelMoney">
            <property name="geometry">
             <rect>
              <x>700</x>
              <y>30</y>
              <width>141</width>
              <height>71</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 28pt &quot;Comic Sans MS&quot;;
color: rgb(248, 73, 95);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QPushButton" name="btnGoBuy">
            <property name="geometry">
             <rect>
              <x>890</x>
              <y>30</y>
              <width>201</width>
              <height>71</height>
             </rect>
            </property>
            <property name="text">
             <string>购买</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_10">
            <property name="geometry">
             <rect>
              <x>600</x>
              <y>30</y>
              <width>61</width>
              <height>61</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;宋体&quot;;</string>
            </property>
            <property name="text">
             <string>金额：</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelVipInfo">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>551</width>
              <height>61</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 18pt &quot;宋体&quot;;
color: rgb(101, 201, 207);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
